cmd/server/main.go

```diff
@@ -39,7 +40,10 @@ func main() {
 		PlaylistStorage: playlistrepo.NewMemory(),
 	}
 
-	service := httpapi.NewApi(accountUseCases, entityUseCases, playlistUseCases)
+	linkCheckerChannel := make(chan string)
```

Comment:
Unbuffered channel will block your workers. Is that what you want from this pipeline?

Reply:
And API call.